tests/libvmifact/factory.go

```diff
@@ -146,3 +146,11 @@ func NewWindows(opts ...libvmi.Option) *kvirtv1.VirtualMachineInstance {
 	vmi.Spec.Domain.Firmware = &kvirtv1.Firmware{UUID: WindowsFirmware}
 	return vmi
 }
+
+func WithCirrosRequestedMemory() libvmi.Option {
```

Comment:
At this point in time, I think we can have a single  `WithSmallOSResourceMemory()`

(https://github.com/kubevirt/kubevirt/pull/11757)

Reply:
Currently, I removed this commit. If I'll see it's still needed, I'll add it in a new PR